detox/detox.d.ts

```diff
@@ -446,7 +446,13 @@ declare global {
 
             readonly system: SystemFacade;
 
-            readonly copilot: DetoxCopilotFacade;
+          /**
+           * @deprecated This API is deprecated and will be removed in the next major version.
+           * Please use `pilot` instead of `pilot`.
+           */
+            readonly copilot: PilotFacade;
```

Comment:
this is only type, you need to initialize this as well alongside the `pilot`.

Reply:
also let's add a log warning when this api is being called with deprecation warning.

Reply:
done